### PR TITLE
Sync rhds/odh-manifests with rhds/modelmesh-serving

### PR DESCRIPTION
**Pull request by anishasthana**
Jira: https://issues.redhat.com/browse/RHODS-6136

Signed-off-by: Anish Asthana <anishasthana1@gmail.com>

### CODE REVIEW

**Comment by anishasthana** [comment]
I haven't deployed it yet for testing. In the process of getting started now.

File: manifests/opendatahub/overlays/odh-model-controller/manager/kustomization.yaml [hunk 1]
```diff
@@ -8,13 +8,10 @@ configMapGenerator:
...
-  - servingruntimes_config.yaml
-  name: servingruntimes-config
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
   newName: quay.io/opendatahub/odh-model-controller
-  newTag: v0.9.5-auth
+  newTag: v0.9.3-auth
```
Comment (heyselbi) [inline manifests/opendatahub/overlays/odh-model-controller/manager/kustomization.yaml:17]: Is the tag downgrading from .5 to .3? In upstream (ODH) repo it is ` newTag: v0.9.5-auth`

Comment (anishasthana) [reply manifests/opendatahub/overlays/odh-model-controller/manager/kustomization.yaml:17]: Hey, yep, I'm just getting both repos in sync. This tag doesn't actually matter as it uses a variable to select what image is deployed.

**Review by VedantMahabaleshwarkar — APPROVED** [review]

**Comment by VedantMahabaleshwarkar** [comment]
lgtm

**Pull request merged**